cmd/gpu-feature-discovery/main.go

```diff
@@ -166,7 +170,15 @@ func start(c *cli.Context, cfg *Config) error {
 		}
 		klog.Infof("\nRunning with config:\n%v", string(configJSON))
 
-		manager := resource.NewManager(config)
+		// TODO: If applicable, an nvmllib needs to be passed to the infolib constructor.
+		nvmllib := nvml.New()
+		devicelib := device.New(device.WithNvml(nvmllib))
```

Comment:
Didn't we change this recently to a required parameter (as opposed to an optional one via `WithNvml()`).

Reply:
Yes. I have not yet included that change. I will pull it in.